app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb

```diff
@@ -233,8 +233,10 @@ def self.host_inv_to_ip(inv, hostname = nil)
       require 'ipaddr'
       default_gw = IPAddr.new(default_gw)
 
-      vnics   = inv.fetch_path("config", "network", "consoleVnic")
-      vnics ||= inv.fetch_path("config", "network", "vnic")
+      vnics         = inv.fetch_path("config", "network", "vnic")
+      console_vnics = inv.fetch_path("config", "network", "consoleVnic")
+      vnics.concat(console_vnics) unless console_vnics.nil?
```

Comment:
I think the management network IP address is most likely what users would want to see, this is the address they would have given the VC to add the host.  The existing code was returning the first address it found but its possible to always return the management network address.

The host will always have at least a management vnic but you're right that could probably be cleaned up in case of an error retrieving the data.  Elsewhere in that source file I see this method for accessing the vnic data, would this be preferable?
```
inv = inv.fetch_path('config', 'network')
nics = inv['consoleVnic'] + inv['vnic']
vnics.to_miq_a.each do |vnic|
```